api/Body.json

```diff
@@ -427,10 +427,10 @@
               }
             ],
             "safari": {
-              "version_added": false
```

Comment:
This is `bodyUsed`. It was there from the start, just with `[ImplementedAs=isDisturbed]` which might have misled some previous attempt here.